react/LocationSearch.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { Input, IconSearch, IconClear } from 'vtex.styleguide'
+import { FormattedMessage } from 'react-intl'
+
+import styles from './LocationSearch.css'
+
+const LocationSearch: React.FC = () => {
+  const [inputValue, setInputValue] = useState<string>('')
+
+  const onClear = () => setInputValue('')
+
+  return (
+    <div className={`${styles.locationSearch} w-100`}>
+      <Input
+        label={
+          <FormattedMessage id="place-components.label.autocompleteAddress" />
+        }
+        prefix={<IconSearch color="#134CD8" />}
```

Comment:
How to use a primary color instead?

Reply:
you can use `className="c-action-primary"`

Reply:
Only works when added on a wrapper div, and the div make it vertically unaligned. IMO adding this to the CSS will do no harm, but if there is a simpler way to do this, just tell me.

```
.locationSearch div :global(.vtex__icon-search) {
  display: flex;
  justify-content: center;
  align-items: center;
}
```

Reply:
can you add these styles to the div itself?

Reply:
Yes, I can (I forgot that I should be doing that lol, ty)